test/jdk/javax/swing/JRadioButton/bug4380543.java

```diff
@@ -82,33 +82,30 @@ public void run() {
 }
 
 class testFrame extends JFrame implements ActionListener {
-    JPanel buttonsPanel;
-
-    Map<String, String> lookAndFeelMaps = new HashMap<String, String>();
-    public testFrame() throws InterruptedException {
+    final Map<String, String> lookAndFeelMap = new HashMap<>();
+    public testFrame() {
         initMap();
         initComponents();
     }
 
-    public void initMap()
-    {
-        String sLnF;
-        String sMapKey;
+    public void initMap() {
+        String sLnFClassName;
+        String sLnFName;
         UIManager.LookAndFeelInfo[] lookAndFeel = UIManager.getInstalledLookAndFeels();
         for (UIManager.LookAndFeelInfo look : lookAndFeel) {
 
-            sLnF = look.getClassName();
-            sMapKey = sLnF.substring(sLnF.lastIndexOf(".")+1);
-            sMapKey = sMapKey.replaceAll("LookAndFeel","");
-            sMapKey = sMapKey.trim();
+            sLnFClassName = look.getClassName();
+            sLnFName = sLnFClassName.substring(sLnFClassName.lastIndexOf(".")+1);
+            sLnFName = sLnFName.replaceAll("LookAndFeel","");
+            sLnFName = sLnFName.trim();
 
-            lookAndFeelMaps.put(sMapKey, sLnF);
+            lookAndFeelMap.put(sLnFName, sLnFClassName);
```

Comment:
Why don't you want to use `look.getName()` instead of doing the magic to extract what resembles the name from the class?